library/std/src/keyword_docs.rs

```diff
@@ -1413,9 +1409,7 @@ mod self_upper_keyword {}
 /// [`extern`]: keyword.extern.html
 /// [`mut`]: keyword.mut.html
 /// [`unsafe`]: keyword.unsafe.html
-/// [`drop`]: mem/fn.drop.html
```

Comment:
`drop` and `mem` are both not in scope.
```suggestion
[`drop`]: crate::mem::drop
```